Frends.IMAP.ReadEmail/Frends.IMAP.ReadEmail/IMAPSettings.cs

```diff
@@ -33,6 +33,19 @@ public class IMAPSettings
         [DefaultValue(false)]
         public bool AcceptAllCerts { get; set; }
 
+        /// <summary>
+        /// Should the task save attachments into designated directory?
```

Comment:
Avoid questions in descriptions. Better put something like "Controls whether task will save attachments to designated directory or not."